docs/faqs/max_execution_time.rst

```diff
@@ -8,31 +8,58 @@ To ensure fairness, there is a maximum execution time for each Qiskit Runtime jo
 
 The system limit on the job execution time is 3 hours for a job running on a simulator
 and 8 hours for a job running on a physical system.
 
 Qiskit Runtime on IBM Quantum
------------------------------
+*****************************
 
-The system limit on the job execution time is
+The system limit on the job execution time is described in the following table:
 
 +------------------+--------------+-----------+--------------+-----------+
 |                  | Public Program           | Private Program          |
 +==================+==============+===========+==============+===========+
 |                  | Premium User | Open User | Premium User | Open User |
 +------------------+--------------+-----------+--------------+-----------+
-| Simulated Device | 3h           | 1h        | 3h           |1h         |
+| Simulated Device | 3h           | 1h        | 3h           | N/A       |
 +------------------+--------------+-----------+--------------+-----------+
-| Real Device      | 8h           | 4h        | 8h           |2h         |
+| Real Device      | 8h           | 4h        | 8h           | N/A       |
 +------------------+--------------+-----------+--------------+-----------+
 
 Note that a *premium user* here means a user who has access to backends in providers other than ibm-q/open/main.
 
+Max execution time
+------------------
+
+There are several factors that impact the program's maximum execution time. In addition to the ``max_execution_time`` parameter, there are time limits on programs and sessions.
+
+Sessions
+********
+
+When a session is started, it is assigned a maximum session timeout value.  After the maximum session timeout is reached, the session is permanently closed. The maximum session timeout value is set on the ``max_time`` parameter, which can be greater than the program’s ``max_execution_time``. By default, it is set to the  system limit.
+
+Additionally, there is a 5 minute *interactive* timeout value. If there are no session jobs queued within that window, the session is temporarily deactivated and normal job selection resumes. After the new session becomes inactive, if the job scheduler gets a job from the original session and its maximum timeout value has not been reached, the session is reactivated until its maximum timeout value is reached.
+  
+.. note:: The timer for ``max_time`` is not paused during any temporary deactivation periods. 
+
+Program time limits
+**********************
+
+Different types of programs have their own time limits:
+
+* **Primitives:** The maximum execution time for the Sampler primitive is 10000 seconds (2.78 hours). The maximum execution time for the Estimator primitive is 18000 seconds (5 hours).
+* **Programs:** The maximum execution time for a program is set on its ``max_execution_time`` parameter.
+* **Prototype programs:** The maximum execution time is listed on the [Prototype programs page](https://quantum-computing.ibm.com/services/programs/prototypes).
+
```

Comment:
Since in line 10 we talked about 1) system limit, 2) program limit, I think this section should go after system limit (e.g. after line 39).

Also it should say "Private programs" instead of just "Programs" to match the tabel.